src/hooks/useSwapCurrencyList.ts

```diff
@@ -366,6 +366,8 @@ const useSwapCurrencyList = (
           slowSearch();
         } else {
           await search();
+          setLowLiquidityAssets([]);
+          setHighLiquidityAssets([]);
```

Comment:
This is unrelated to the feature. L2 search results on this feature branch were retaining mainnet's results in categories outside of verified assets. This fixes that.

Reply:
🚀